setup.py

```diff
@@ -16,8 +16,28 @@
 
 setup(name='haas',
       version='0.2rc2',
+      author='https://github.com/CCI-MOC/haas/network/members',
+      author_email = 'haas-dev-list@bu.edu',
```

Comment:
docs - bottom of the page https://docs.python.org/2/distutils/setupscript.html. Ian is right about maintainer and maintainer_email.